minicluster/src/main/java/org/apache/accumulo/cluster/standalone/StandaloneAccumuloCluster.java

```diff
@@ -184,4 +188,12 @@ public ClusterUser getUser(int offset) {
     checkArgument(offset >= 0 && offset < users.size(), "Invalid offset, should be non-negative and less than " + users.size());
     return users.get(offset);
   }
+
+  @Override
+  public AccumuloConfiguration getSiteConfiguration() {
+    Configuration conf = new Configuration(false);
+    Path accumuloSite = new Path(serverAccumuloConfDir, "accumulo-site.xml");
```

Comment:
I Am not sure what the current code will do in that case.  Personally I think it should fail here, instead of later where it make it harder to debug the issue.